rts/lra/client/src/main/resources/META-INF/services/javax.ws.rs.client.ClientBuilder

```diff
@@ -1 +1,2 @@
-org.jboss.resteasy.microprofile.client.impl.MpClientBuilderImpl
+org.jboss.resteasy.client.jaxrs.ResteasyClientBuilder
```

Comment:
I get `java.lang.ClassNotFoundException: org.glassfish.jersey.client.JerseyClientBuilder` when I use the client straight from the code base. IIRC `org.glassfish.jersey.client.JerseyClientBuilder` is the last attempt to find a ClientBuilder implementation in the JAX-RS specification

Reply:
Okay, so your reproducer in the previous comment is such an example and is a good reason to include it. I'd still like to get some clarification on my question below about whether we need to tell the service loader about both classes, in particular if we can get rid of the most derived class (MpClientBuilderImpl) then that would be beneficial since it would remove the dependency on whatever library is providing MpClientBuilderImpl.

Reply:
I'll dive deeper into this and I'll report back :+1:

Reply:
What happens if you just remove `org.jboss.resteasy.microprofile.client.impl.MpClientBuilderImpl`. Does the TCK and other tests still pass.

Reply:
If you check the git commit history the commit comment (and associated JIRA) may indicate the rationale for replacing ResteasyClientBuilder with the more specific class.

Reply:
Thanks for checking this, even though the TCK still works, then as you reported it was need by a community member when he was testing against quarkus (https://issues.redhat.com/browse/JBTM-3381).

I willl mark this conversation as resolved.

Reply:
I can confirm that `MpClientBuilderImpl` was added because of Quarkus-based applications. We'll keep `MpClientBuilderImpl` among possible `ClientBuilder` implementations to allow Quarkus users to use lra-client out of the box.